### PR TITLE
TOOL-30 Updated gems related methods to check for gems.locked file too.

### PR DESCRIPTION
**Pull request by richard-bogdan-bitrise**

### CODE REVIEW

File: command/gems/gemfile_lock.go [hunk 1]
```diff
@@ -86,3 +89,41 @@ func ParseBundlerVersion(gemfileLockContent string) (gemVersion Version, err err
...
 	}, nil
 }
+
+var gemFileLockNames = []string{
+	"Gemfile.lock", "gems.locked",
+}
+
+// GemFileLockPth gets the path for the gem lock file from the given directory.
+func GemFileLockPth(searchDir string) (string, error) {
+	var pth string
```
Comment (lpusok) [inline command/gems/gemfile_lock.go]: Would use a setup like this:
```
	for i, gemFileName := range gemFileLockNames {
		pth := filepath.Join(searchDir, gemFileName)
		if _, err := os.Stat(pth); err != nil {
			if os.IsNotExist(err) {
				continue
			}
			return "", err
		}
		return pth, nil
	}
	return "", errors.New("no lock file found")
```

File: command/gems/gemfile_lock.go [hunk 2]
```diff
@@ -86,3 +89,41 @@ func ParseBundlerVersion(gemfileLockContent string) (gemVersion Version, err err
...
+			return "", err
+		} else if err == nil {
+			return pth, nil
+		}
+	}
+	return pth, nil
+}
+
+// GemFileLockContent gets the content of the gem lock file from the given directory.
+func GemFileLockContent(searchDir string) (string, error) {
```
Comment (lpusok) [inline command/gems/gemfile_lock.go]: Not entirely convinced that GemFileLockContent and ParseVersionFromBundlePth are helpful enough to be added as a package level exported functions.

Comment (richard-bogdan-bitrise) [reply command/gems/gemfile_lock.go]: Would be needed for fastlane step.

File: command/gems/gemfile_lock.go (hunk 1 above)
Comment (richard-bogdan-bitrise) [reply command/gems/gemfile_lock.go]: Updated code.

**Comment by richard-bogdan-bitrise** [comment]
@godrei pls merge.

**Pull request merged**